eyeshade/controllers/accounts.js

```diff
@@ -24,26 +24,12 @@ const joiBAT = braveJoi.string().numeric()
 
 const selectAccountBalances = `
 SELECT
-  q.account_id,
-  q.account_type,
-  coalesce(sum(q.amount), 0.0) as balance
-FROM (
-  SELECT
-    from_account AS account_id,
-    from_account_type as account_type,
-    (0 - amount) AS amount
-  FROM transactions
-  WHERE from_account = any($1::text[])
-  UNION
-  SELECT
-    to_account AS account_id,
-    to_account_type as account_type,
-    sum(amount) AS amount
-  FROM transactions
-  WHERE to_account = any($1::text[])
-  GROUP BY to_account_type, to_account
-) AS q
-GROUP BY (q.account_type, q.account_id)
+  account_transactions.account_type as account_type,
+  account_transactions.account_id as account_id,
+  COALESCE(SUM(account_transactions.amount), 0.0) as balance
+FROM account_transactions
+WHERE account_id = any($1::text[])
+GROUP BY (account_transactions.account_id, account_transactions.account_type);
```

Comment:
I know that logically this should be the same but can we reverse this like so? Just to maintain the same structure as the first materialized view.

```suggestion
GROUP BY (account_transactions.account_type, account_transactions.account_id);
```